addons/src/createAutoCorrectedDatePipe.js

```diff
@@ -1,9 +1,12 @@
-export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy') {
+export default function createAutoCorrectedDatePipe(dateFormat = 'mm dd yyyy', {
+  minYear = 1,
+  maxYear = 9999
+} = {}) {
   return function(conformedValue) {
     const indexesOfPipedChars = []
     const dateFormatArray = dateFormat.split(/[^dmyHMS]+/)
-    const maxValue = {'dd': 31, 'mm': 12, 'yy': 99, 'yyyy': 9999, 'HH': 23, 'MM': 59, 'SS': 59}
-    const minValue = {'dd': 1, 'mm': 1, 'yy': 0, 'yyyy': 1, 'HH': 0, 'MM': 0, 'SS': 0}
```

Comment:
@dclark27 With this now set to `1` it has highlighted a bug: you can't type numbers below `1000`, for example `0900` - you can't type the first zero. you have to type a number greater than 1, then move the cursor back to type the 0.. 

Not really sure how much this matters.. I mean who would use dates below 1000.. however, it does break backwards compatibility.  Is there any way to cover that case?

At a minimum you could auto-disable the feature if the min and max values are the defaults.. that would at least preserve backwards compatibility..

Reply:
I played around with this a bit. If we could change the default minYear to 0001 instead of 1 we'd be in good shape but that breaks eslint rules. Instead, I'll go with your idea to skip the feature if min and max aren't specified.